src/main/java/com/akademia/educators_back/mapper/CommentMapper.java

```diff
@@ -0,0 +1,27 @@
+package com.akademia.educators_back.mapper;
+
+import com.akademia.educators_back.to.CommentTo;
+import com.akademia.educators_back.entity.CommentEntity;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CommentMapper {
+
+    public CommentEntity toCommentEntity(CommentTo commentTo){
+        CommentEntity commentEntity = new CommentEntity();
+        if(commentTo.getId() != null){
+            commentEntity.setId(commentTo.getId());
+        }
+        commentEntity.setAnswer(commentTo.getAnswer());
+        return commentEntity;
+    }
+
+    public CommentTo toCommentTO(CommentEntity commentEntity){
+        CommentTo commentTo = new CommentTo();
+        if(commentEntity.getId() != null){
```

Comment:
Czy id encji może być nullem skoro wyciągamy ją z bazy? Jeśli już to sprawdziłbym czy sama encja nie została przypadkiem usunięta. Zadanie z mapperem powinno iść osobnym branchem, żeby nie mieszać za dużo przy okazji jednego zadania :)